src/webui/webapplication.h

```diff
@@ -177,6 +177,7 @@ class WebApplication final : public ApplicationComponent<QObject>
         {{u"torrents"_s, u"addPeers"_s}, Http::METHOD_POST},
         {{u"torrents"_s, u"addTags"_s}, Http::METHOD_POST},
         {{u"torrents"_s, u"addTrackers"_s}, Http::METHOD_POST},
+        {{u"torrents"_s, u"addWebseeds"_s}, Http::METHOD_POST},
```

Comment:
I would prefer if it is written as `addWebSeed` (and other endpoints accordingly).

Reply:
You can see from the code directly above that existing endpoints use the plural form.

Reply:
> You can see from the code directly above that existing endpoints use the plural form.

Sorry. I missed `s` at the end. I mean `addWebSeeds`.

Reply:
I missed that this was about capitalization. Will update.